admin/src/js/controllers/edit-user.js

```diff
@@ -53,6 +53,10 @@ angular
       });
     };
 
+    const formatDate = (date) => {
+      return moment(date).format('DD-MMM-YYYY HH:mm:ss');
+    };
```

Comment:
We can still use the configured date format, and it's really easy because the calling code already has the `settings`.

```suggestion
    const formatDate = (settings, date) => {
      const format = settings.reported_date_format || 'DD-MMM-YYYY HH:mm:ss';
      return moment(date).format(format);
    };
```